Web.config

```
@@ -6,8 +6,11 @@
   -->
 <configuration>
 	<connectionStrings>
-		<add name="con" connectionString="Data Source=LAPTOP-AIAUKSVV\SQLEXPRESS;Initial Catalog=LogBook'Integrated Security=true"/>
-	</connectionStrings>
+  <add name="con" connectionString="Data Source=.\SQLEXPRESS;Initial Catalog=LogBook;User ID=sa;Password=experior12345;Trusted_Connection=False;"
+   providerName="System.Data.SqlClient" />
+  <add name="LogBookConnectionString" connectionString="Data Source=LAPTOP-AIAUKSVV\SQLEXPRESS;Initial Catalog=LogBook;Integrated Security=True"
```

No reason to give the name _LogBookConnectionString_

Give it a proper name to refer to your DB. The shorter and more readable, the better your code will be. 
```suggestion
  <add name="LogBookDB" connectionString="Data Source=LAPTOP-AIAUKSVV\SQLEXPRESS;Initial Catalog=LogBook;Integrated Security=True"
```

Of course, if you change this name, you will need to change the names of all the connection strings in your aspx.cs files.